### DISCUSSION TITLE
Setting up a personal server

### DISCUSSION THREAD

**Discussion by mahdisarlak** [vpnhood/VpnHood#163] [Q&A] [answered]
First of all, thank you for the amazing and professional development of VPNhood

Hello, good time, I have no technical knowledge about setting up a private server, I have bought a private server, but I don't know how to install VPNhood on it, if there is a video tutorial for setting up and installing, please send me

**Comment by trudyhood** [comment, marked as answer]
Hi, Thank you for your interest in VpnHood. 

We hope you can install it without any problem. We try to make sure everyone can install it. I recommend you use our VpnHood Access Server; it helps you to install and manage your client. 

Go to the following page and let us know if you face any issues. Please don't hesitate to share your experience; we can improve the document and will try to solve any bug or issue you may encounter. If it doesn't work, we can help you to install your server, it also help us to solve any issue that Server Owner may have.
https://github.com/vpnhood/VpnHood/wiki/VpnHood-Access-Server

**Comment by mahdisarlak** [comment]
Are you installing it on my server now?
How can I get your help in this matter?

**Reply by bobvhood** [reply]
Please follow the instruction and let us know the part you can't understand or need help
https://github.com/vpnhood/VpnHood/wiki/VpnHood-Access-Server
